rosetta-source/src/main/rosetta/product-common-settlement-type.rosetta

```diff
@@ -367,10 +376,15 @@ type PrincipalPayment: <"Any kind of principal payments when the amount is known
         if presentValuePrincipalAmount exists
         then discountFactor exists
 
+type CashOrPhysicalSplitRatio: <"When setllementType is CashOrPhysical, the percentage split between cash and physical to apply on total deliverable quantity.">
+    cashRatio number (1..1) <"Cash ratio to apply on total deliverable quantity, for instance 80%, When setllementType is CashOrPhysical.">
+    physicalRatio number (1..1) <"Physical ratioto apply on total deliverable quantity, for instance 20%, When setllementType is CashOrPhysical.">
```

Comment:
Shouldn't you have a condition that
```
cashRatio + physicalRatio = 1.0
```
?

Reply:
yes, agreed, just do not know how to properly draft such kind of condition, kindly help / draft it, please